src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/content-browser/content-browser.component.ts

```diff
@@ -38,11 +37,7 @@ export class ContentBrowserComponent extends BaseBrowserComponent<Content> imple
     super('ContentBrowserComponent', injector, facade);
   }
 
-  async init(): Promise<void> {
-    this.sub = this.basketService.basketChange.subscribe(basketItems => {
-      this.checkItemsInBasket(basketItems);
-    });
-  }
+  async init(): Promise<void> {}
```

Comment:
Do we need this empty method?

Reply:
It's an abstract method so I cannot remove it.

Reply:
I didn't realize that. Thanks